app/views/webauthn_verifications/prompt.html.erb

```diff
@@ -1,16 +1,23 @@
 <% @title = t(".title")%>
 
 <% if @user.webauthn_credentials.any? %>
+  <% browser = Browser.new(request.env["HTTP_USER_AGENT"])  %>
   <div class="t-body">
+    <% if browser.safari? %>
+      <p class="l-text-red-600" ><%= t(".safari_note") %></p>
+    <% end %>
     <h2><%= t(".authenticating_as") %> <%= @user.name %></h2>
     <p><%= t("settings.edit.webauthn_credential_note") %></p>
   </div>
 
   <%= form_tag authenticate_webauthn_verification_path, method: :post, class: "js-webauthn-session-cli--form", data: { options: @webauthn_options.to_json } do %>
     <div class="form_bottom">
       <p hidden class="l-text-red-600 js-webauthn-session-cli--error"></p>
-
-      <%= submit_tag t(".authenticate"), class: 'js-webauthn-session-cli--submit form__submit form__submit--no-hover' %>
+      <% if browser.safari? %>
+        <%= submit_tag t(".authenticate"), class: 'js-webauthn-session-cli--submit form__submit form__submit--no-hover', disabled: true %>
```

Comment:
could remove the duplication with `disabled: browser.safari?`

Reply:
Good suggestion, addressed in latest commit.